### ISSUE TITLE
NaN errors occurring constantly

### ISSUE THREAD

**Issue by wdsongama** [muslll/neosr#83]
Hi! I am using the latest version of neosr, and I train my models using Anaconda. I made sure to install all the requirements and have my drivers up to date. But I really can't pinpoint what is causing NaN error. I like to mess around with values and options, so if could list what can't/shoundn't be changed in the script, that would be nice... I like to experiment and learn, so maybe I am doing a newbie mistake that you could easily identify. I am using "train_esrgan.toml" and training from scratch.
[train_esrgan.txt](https://github.com/user-attachments/files/16519235/train_esrgan.txt)

**Comment by muslll** [comment]
Hi. There's couple of issues on your configuration:
- The most important one is not using a pretrained model. It is highly recommended that you do so using [`pretrain_network_g`](https://github.com/muslll/neosr/wiki/Configuration-Walkthrough#path), otherwise you will very likely have NaNs due to running with GAN and adan_sf from scratch. You can find pretrains [here](https://openmodeldb.info/).
- Learning rate is too low for the generator. Try using at least `5e-4`, ideally use the defaults.
- EMA is disabled. It is recommended that you enable with while using `adan_sf`
- Some values on `optim_d` are commented out. Use default values.
- GAN loss weight is too high, use defaults

The best way to ensure training works is by using the default configuration templates and a pretrained model. Then after you get it working you can start to modify things to your preference.
